config/swagger.yaml

```diff
@@ -3305,37 +3133,21 @@ definitions:
           contract:
             $ref: '#/definitions/EncodedPubkey'
             description: Contract address
-          # vm_version kept for backwards compatibility, should be removed
-          vm_version:
-            type: integer
-            description: ABI version for the call/calldata
-            minimum: 0
-            maximum: 65535
           abi_version:
-            type: integer
-            description: ABI version for the call/calldata
-            minimum: 0
-            maximum: 65535
+            $ref: '#/definitions/UInt16'
           amount:
-            type: integer
-            description: Amount of tokens to transfer to the contract
-            format: int64
+            $ref: '#/definitions/UInt'
           gas:
-            type: integer
-            description: Gas limit for the contract call
-            format: int64
+            $ref: '#/definitions/UInt64'
           gas_price:
-            type: integer
-            description: Gas price for the contract call
-            format: int64
+            $ref: '#/definitions/UInt'
           call_data:
             description: Contract call data
             $ref: '#/definitions/EncodedByteArray'
         required:
           - caller
           - contract
-          # should be required once vm_version is removed
-          # - abi_version
+          - abi_version
```

Comment:
I see this covered by https://github.com/aeternity/aeternity/blob/1c6a34e0c4f940ce3c9d10ac0f465b9b07ecc57f/docs/release-notes/next-fortuna/PT-164962293-no_more_backwards_compat_abi_and_vm.md
Ok.